### PR TITLE
add new automated tests for several modules (1)

### PR DESCRIPTION
**Pull request by tedstrauss**

### CODE REVIEW

File: modules/candidate_parameters/test/candidate_parametersTest.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+     * Tests that, when loading the candidate_parameters module, some
+     * text appears in the body.
+     *
+     * @return void
+     */
+    function testCandidateParametersDoespageLoad()
+    {
+        $this->webDriver->get($this->url . "?test_name=candidate_parameters");
+        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
+        $this->assertContains("Candidate Parameters", $bodyText);
```
Comment (driusan) [inline modules/candidate_parameters/test/candidate_parametersTest.php]: missing subtests: add_family, update_participant_status, update_candidate_info, update_proband_info, and update_consent_info

File: modules/dicom_archive/test/dicom_archiveTest.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+     * Tests that, when loading the dicom_archive module, some
+     * text appears in the body.
+     *
+     * @return void
+     */
+    function testdicomArchiveDoespageLoad()
+    {
+        $this->webDriver->get($this->url . "?test_name=dicom_archive");
+        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
+        $this->assertContains("Dicom Archive", $bodyText);
```
Comment (driusan) [inline modules/dicom_archive/test/dicom_archiveTest.php]: missing viewDetails subtest

File: modules/final_radiological_review/test/final_radiological_reviewTest.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+     * Tests that, when loading the final_radiological_review module, some
+     * text appears in the body.
+     *
+     * @return void
+     */
+    function testFinalRadiologicalReviewDoespageLoad()
+    {
+        $this->webDriver->get($this->url . "?test_name=final_radiological_review");
+        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
+        $this->assertContains("Final Radiological Review", $bodyText);
```
Comment (driusan) [inline modules/final_radiological_review/test/final_radiological_reviewTest.php]: missing test_name=final_radiological_review&subtest=final_radiological_review subtest

File: modules/candidate_list/test/candidate_listTest.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+     *
+     * @return void
+     */
+    function testCandidateListDoespageLoad()
+    {
+        $this->webDriver->get($this->url . "?test_name=candidate_list");
+        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
+        $this->assertContains("Access Profile", $bodyText);
+    }
+}
```
Comment (driusan) [inline modules/candidate_list/test/candidate_listTest.php]: Tara's pull request deletes the certification module. I don't think we should include this test.

File: modules/genomic_browser/test/genomic_browserTest.php [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+     * text appears in the body.
+     *
+     * @return void
+     */
+    function testGenomicBrowserDoespageLoad()
+    {
+        $this->webDriver->get($this->url . "?test_name=genomic_browser");
+        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
+        $this->assertContains("Genomic Browser", $bodyText);
+    }
```
Comment (driusan) [inline modules/genomic_browser/test/genomic_browserTest.php]: missing snp_browser submenu

File: modules/imaging_browser/test/imaging_browserTest.php [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+     * Tests that, when loading the imaging_browser module, some
+     * text appears in the body.
+     *
+     * @return void
+     */
+    function testImagingBrowserDoespageLoad()
+    {
+        $this->webDriver->get($this->url . "?test_name=imaging_browser");
+        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
+        $this->assertContains("Imaging Browser", $bodyText);
```
Comment (driusan) [inline modules/imaging_browser/test/imaging_browserTest.php]: missing viewSession subtest

File: modules/help_editor/test/help_editorTest.php [hunk 7]
```diff
@@ -0,0 +1,43 @@
+help_editor
```
Comment (driusan) [inline modules/help_editor/test/help_editorTest.php]: I don't think this line is supposed to be here

File: .travis.yml [hunk 8]
```diff
@@ -75,20 +75,39 @@ script:
...
+    - vendor/bin/phpunit modules/instrument_builder/test/
+
+    - vendor/bin/phpunit modules/brainbrowser/test
+    - vendor/bin/phpunit modules/candidate_list/test
+    - vendor/bin/phpunit modules/candidate_parameters/test
+    - vendor/bin/phpunit modules/certification/test
+    - vendor/bin/phpunit modules/configuration/test
+    - vendor/bin/phpunit modules/conflict_resolver/test
+    - vendor/bin/phpunit modules/create_timepoint/test
+    - vendor/bin/phpunit modules/data_integrity_flag/test
```
Comment (driusan) [inline .travis.yml]: The data integrity file test is in: modules/data_integrity_flag/tests/data_integrity_flagTest.php in this pull request, (not test), so this is causing it to fail.

File: modules/final_radiological_review/test/final_radiological_reviewTest.php (hunk 3 above)
Comment (tedstrauss) [inline modules/final_radiological_review/test/final_radiological_reviewTest.php]: the test and subtest names are the same?? sure you got that one right?

Comment (driusan) [inline modules/final_radiological_review/test/final_radiological_reviewTest.php]: Yes. If there's no subtest, it loads the menu filter, and if there is a subtest, it loads the form that allows you to edit the radiological review.

We probably can/should rename the subtest, but right now it's named the same thing.

**Pull request merged**